### DISCUSSION TITLE
First time contributer that submited a security issue and would like to propose fix shall open discussion as well ?

### DISCUSSION THREAD

**Discussion by flegare** [open-webui/open-webui#3462] [Q&A]
Process indicate to open a discussion prior doing a pull request. Does this apply to security fix as well?

**Comment by justinh-rahb** [comment]
If you feel it's a sensitive matter, you can use the GH security disclosure form: https://github.com/open-webui/open-webui/security/advisories/new

**Reply by flegare** [reply]
I did, but the question is more after the submission via this form.  

If I propose a pull request I suspect I can't refer directly to the issue in discussion which I suspect is less open to the public.
